packages/flutter/test/cupertino/context_menu_test.dart

```diff
@@ -197,6 +240,84 @@ void main() {
       expect(findStatic(), findsOneWidget);
     });
 
+    testWidgets('CupertinoContextMenu with a basic builder opens and closes the same as when providing a child', (WidgetTester tester) async {
```

Comment:
If we deprecate or remove `child` as mentioned above, maybe change the name of this test.